pupil_src/launchables/eye.py

```diff
@@ -590,7 +591,55 @@ def window_should_update():
                     and notification["target"] == g_pool.process
                 ):
                     replace_source(notification["name"], notification["args"])
+                elif notification["subject"].startswith("pupil_detector.set_property"):
```

Comment:
Do you think you could extract the notification handling to some other method? I count a maximum of ten levels of indentations, which I feel are a bit too many.

Reply:
I do not think this is easily possible and deserves its own issue/pr.